src/librustc/ty/layout.rs

```diff
@@ -1003,6 +1061,16 @@ impl<'a, 'gcx, 'tcx> Layout {
                     }
                 }
 
+                if def.variants.len() == 1 && hint == attr::ReprAny{
```

Comment:
Missing space before `{`. Also, this will never be reached, maybe you meant to put the `hint == attr::ReprAny` in the previous `if`?